src/D2L.Bmx/ConfigureHandler.cs

```diff
@@ -4,19 +4,26 @@ internal class ConfigureHandler( IBmxConfigProvider configProvider, IConsoleProm
 	public void Handle(
 		string? org,
 		string? user,
-		int? duration
+		int? duration,
+		bool nonInteractive
 	) {
 
 		if( string.IsNullOrEmpty( org ) ) {
+			if( nonInteractive ) {
```

Comment:
Note: in other similar areas we use `if( !nonInteractive ) {` conditions to control flow but:
* I don't like double negatives in code when they can be avoided
* I think this is clearer anyways

Happy to switch it back (or update the other spots)

Reply:
I like this style better in general.

That said, I had a different design in mind when I made that card:
If `--non-interactive` is specified and a config property isn't specified on the command line, it will simply be omitted from the config file.

So I'd want this instead:
```csharp
if( string.IsNullOrEmpty( org ) && !nonInteractive ) {
	org = consolePrompter.PromptOrg();
}
```
No error.
`BmxConfigProvider.SaveConfiguration` will skip writing any properties that have a null/default/empty value.

Does this sound reasonable?

Reply:
Yup - makes sense and sounds good.
I guess the other commands literally don't work without org/user, whereas this one can still do part of the work.